itou/users/models.py

```diff
@@ -134,6 +135,25 @@ def get_duplicates_by_pole_emploi_id(self, prefetch_related_lookups=None):
 
         return result
 
+    def job_seekers_ids_in_user_list(self, user, organization=None):
```

Comment:
```suggestion
    def linked_job_seekers(self, user, organization=None):
```
?
Avec éventuellement une docstring: """Job seekers that appear in the user's job seekers list view""" ?

Reply:
Et en fait, ça serait plutôt `linked_job_seeker_ids` (je n'avais plus en tête que le `queryset` envoyait une `UNION` d'`id`)